region/src/main/java/org/motechproject/nms/region/service/CensusVillageImportService.java

```diff
@@ -1,4 +1,7 @@
 package org.motechproject.nms.region.service;
 
+import org.motechproject.nms.region.domain.District;
+
 public interface CensusVillageImportService extends LocationDataImportService {
+    void addParent(District district);
```

Comment:
Aren't all those addParent(..) methods supposed to be called from LocationDataImportController too?